Editor/Resources/EditorWindow/Pages/ManagedEC2Page.uxml

```diff
@@ -0,0 +1,46 @@
+﻿<!-- Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. -->
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
+<ui:UXML xmlns:ui="UnityEngine.UIElements">
+    <Style src="../common.uss"/>
+    <ui:VisualElement class="separator separator--large">
+        <ui:VisualElement class="separator">
+            <ui:Label name="ManagedEC2Title" class="page__title" text="Managed EC2"/>
+            <ui:Label name="ManagedEC2Description" text="Managed EC2 fleets use Amazon EC2 instances to host your game servers.&#10;Amazon GameLift manages the instances and removes the burden of hardware and software management from hosting your games."/>
+        </ui:VisualElement>
+        <ui:VisualElement class="divider"/>
+        <ui:Foldout name="ManagedEC2IntegrateTitle" class="foldout separator separator--small" text="Integrate GameLift with your game client">
+            <ui:Label name="ManagedEC2IntegrateDescription" text="Add the Amazon GameLift server SDK to your game server and add code to communicate with the GameLift hosting service. If you&apos;re working with the Amazon GameLift sample game, this integration is already done."/>
+            <ui:VisualElement class="external-link">
+                <ui:Label name="ManagedEC2IntegrateLink" text="Setup Amazon GameLift on a backend service"/>
+                <ui:Image name="ExternalLink" class="icon"/>
+        </ui:VisualElement>
+        </ui:Foldout>
+        <ui:Foldout name="ManagedEC2ScenarioTitle" class="foldout" text="Select Deployment Scenario"/>
+        <ui:Foldout name="ManagedEC2ParametersTitle" class="foldout" text="[GameName] parameters"/>
```

Comment:
Should the element name here be `ManagedEC2ParametersFoldout` instead?

Reply:
I can change it if needed, I had it as title as it does act as the title for that section